pkg/util/ipvs/ipvs.go

```diff
@@ -110,3 +122,29 @@ func (rs *RealServer) Equal(other *RealServer) bool {
 	return rs.Address.Equal(other.Address) &&
 		rs.Port == other.Port
 }
+
+func GetKernelVersionAndIPVSMods(Executor exec.Interface) (kernelVersion string, ipvsModules []string, err error) {
+	// get kernel version
+	kernelVersionFile := "/proc/sys/kernel/osrelease"
+	out, err := Executor.Command("cut", "-f1", "-d", " ", kernelVersionFile).CombinedOutput()
+	if err != nil {
+		return "", nil, fmt.Errorf("error getting os release kernel version: %v(%s)", err, out)
+	}
+	kernelVersion = strings.TrimSpace(string(out))
+
+	// get ipvs modules
+	ver1, err := version.ParseGeneric(kernelVersion)
```

Comment:
nit: You are parsing kernel version here so I assume you can move this line upper.

Reply:
this line is for comparing versions (the kernel version the function returns is got in Line 133)

Reply:
I mean the comment is not consistent with the code. Comment says get ipvs modules but you are parse kernel verison.